contributors/devel/sig-architecture/feature-gates.md

```diff
@@ -0,0 +1,61 @@
+# Feature gate progression in Kubernetes
+
+Feature flags are how Kubernetes tracks progression of functionality that is available when deployed.
+
+## Lifecycle
+
+Feature progress through `Alpha` -> `Beta` -> `GA`. Sometimes we end up deciding that a feature is not going to be supported and we end up marking them as `Deprecated`.
```

Comment:
Something like "Most features will go through all three stages, but occasionally there are features which can skip some." ?

Reply:
makes sense to me.